dotcom-rendering/src/web/components/SlotBodyEnd/BrazeEpic.tsx

```diff
@@ -33,7 +34,7 @@ type EpicConfig = {
 export const canShowBrazeEpic = async (
 	brazeMessagesPromise: Promise<BrazeMessagesInterface>,
 	brazeArticleContext: BrazeArticleContext,
-): Promise<CanShowResult<any>> => {
+): Promise<CanShowResult<Meta>> => {
```

Comment:
Assuming this type is a container for meta data here?

Reply:
Yeah exactly. The overall return value is a `Promise` which resolves with a `CanShowResult` which is parameterised with `Meta`. `CanShowResult` looks like this:

```
export type CanShowResult<T> = ShouldShow<T> | ShouldNotShow;
```

So `ShouldShow` is generic over `T`, which in this case we make concrete with `Meta`.